spec/requests/api/create_article_spec.rb

```diff
@@ -63,5 +72,21 @@
         expect(response_json['errors']).to eq("Body can't be blank")
       end
     end
+
+    describe 'because the category is missing' do
+      before do
+        post '/api/articles',
+             params: { article: { title: 'I forgot the category',
+                                  lede: 'I forgot the category',
+                                  body: 'I forgot the category',
+                                  category_name: category.name } }
+      end
+
+      it { is_expected.to have_http_status 422 }
+
+      it 'is expected to as for the category when the category is missing' do
+        expect(response_json['errors']).to eq("Category can't be blank and Category must exist")
+      end
+    end
```

Comment:
yeah.. you see, category is not missing here)) we have category name and theoretically it is enough to create this article becouse API knows category id if it knows category name.

Reply:
Got it. We've changed our implementation to have the API deal with finding the correct `category_id`.